integrations/googlecalendar/integration.definition.ts

```diff
@@ -77,3 +96,18 @@ export default new IntegrationDefinition({
     },
   },
 })
+  .extend(interfaces.listable, (self) => ({
+    item: self.event,
+  }))
+  .extend(interfaces.creatable, (self) => ({
```

Comment:
nitpick: I cant comment on the exact line since it wasn't changed, the event output schemas don't validate dates using z.string().date() it validates ISO 8601 dates so [it should work](https://stackoverflow.com/questions/522251/whats-the-difference-between-iso-8601-and-rfc-3339-date-formats) on rfc 3339 but requires testing